portfolio/src/main/webapp/script.js

```diff
@@ -35,6 +35,34 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+//*********************** PAGE HANDLING ***********************
+/**
+ * Displays the right sidebar
+ */
+let isSidebarOpen = false;
+function toggleSidebar(){
+    let button = document.getElementById("commentbutton");
+    let el = document.getElementById("sidebar");
+    if (isSidebarOpen == true){
+        closeSidebar(el, button);
+    } else {
+        openSidebar(el, button);
+    }
+    isSidebarOpen = !isSidebarOpen;
+}
+
+function openSidebar(el, button){
+    el.style.width = "0px";
```

Comment:
This seems backwards. Wouldn't you want to set the width to 550px when you open the sidebar?

Reply:
I see what you mean...Fn names were referring to the current state of the sidebar.
Also need to change in my other commit as well as the jsdoc files i was lazy when making them